.dockerignore

```diff
@@ -0,0 +1,10 @@
+Dockerfile*
+docker-compose*
+.dockerignore
+.git
+.gitignore
+README.md
+LICENSE
+.vscode
+.idea
+venv
```

Comment:
I have `.pytest_cache` and `mypy_cache` lying around and I'm probably not the only one. Those make a good addition to this list.